src/components/Icons/IconDialog/IconDialog.tsx

```diff
@@ -35,7 +35,7 @@ export const IconDialog: React.FC<IconDialogProps> = ({icon, onClose, onClickToK
     }
 
     return (
-        <Dialog className={b()} size="s" open={true} onClose={onClose}>
+        <Dialog className={b()} modalClassName={b('modal')} size="s" open={true} onClose={onClose}>
```

Comment:
and this

Reply:
Hm... I checked again. We don't need to set border-radius, because it's already 24px.
For some reason, when updating uikit to new major border-radius was less than 24px